copper-server/src/main/java/com/pogeyan/cmis/browser/AkkaServletContextListener.java

```diff
@@ -98,11 +97,14 @@
 	private static final String DEFAULT_DB_CLIENT_FACTORY = "com.pogeyan.cmis.data.mongo.services.MongoClientFactory";
 
 	static final Logger LOG = LoggerFactory.getLogger(AkkaServletContextListener.class);
+	static ActorServiceFactory sf;
+	static {
+		sf = new ActorServiceFactory();
+	}
```

Comment:
why initializing here? ActorServiceFactory should have a "getInstance" method that returns the singleton from the class itself. Please change it.

Reply:
added the getInsatance method